server/tests-py/queries/remote_schemas/add_remote_schema_err_missing_arg.yaml

```diff
@@ -11,6 +11,6 @@ query:
   args:
     name: err-missing-arg
     definition:
-      url: http://localhost:5000/iface-graphql-err-missing-arg
+      url: ${REMOTE_GRAPHQL_ROOT_URL}/iface-graphql-err-missing-arg
```

Comment:
Take the root url of remote GraphQL server from the environmental variable ${REMOTE_GRAPHQL_ROOT_URL}  instead. This will be set by the remote_gql_server fixture when the remote server is running.